gnovm/pkg/gnolang/machine.go

```diff
@@ -994,12 +995,52 @@ const (
 	OpCPUReturnCallDefers  = 1
 )
 
+type Executable interface {
+	Exec(m *Machine, o Op) (int64, error)
+}
+
 //----------------------------------------
 // main run loop.
 
+func (m *Machine) doExec(op Op) bool {
```

Comment:
this method will eventually disappear. It is made to make it compatible with old behavior.